docs /zh-cn /README.md

```diff
@@ -1,7 +1,6 @@
 # QMK机械键盘固件
```

Comment:
This file should be renamed back, removing the added space.

Reply:
Not sure how that happened ill try to fix it